e2e/test_teams.py

```diff
@@ -128,6 +129,7 @@ def test_api_v0_update_team(team):
     assert data['slack_channel'] == slack
     assert data['slack_channel_notifications'] == slack_notifications
     assert data['override_phone_number'] == override_num
+    assert data['api_managed_roster'] == True
```

Comment:
Nit/flake8 may complain: `s/== True/is True/g`

Reply:
fixed